crates/tauri-runtime-wry/Cargo.toml

```diff
@@ -17,7 +17,7 @@ rustc-args = ["--cfg", "docsrs"]
 rustdoc-args = ["--cfg", "docsrs"]
 
 [dependencies]
-wry = { version = "0.43.1", default-features = false, features = [
```

Comment:
waiting on https://github.com/tauri-apps/wry/pull/1361 and the next release of wry